FreeRTOS/FreeRTOS-Kernel
# Add the formatting bot action to FreeRTOS-Kernel

Description
-----------
Add the formatting bot action to FreeRTOS-Kernel

Test Steps
-----------
Bot can be seen working in FreeRTOS-Plus-TCP: [Here](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/pull/1017#issuecomment-1707462829)

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
